infracost-usage-example.yml

```diff
@@ -874,3 +874,8 @@ resource_usage:
 
   azurerm_notification_hub_namespace.my_namespace:
     monthly_pushes: 1000000 # Monthly total number number of additional pushes.
+
+  azurerm_virtual_network_gateway.Basic:
+    s2s_tunnel: 12 # Total number of s2s tunnels.
```

Comment:
I'm not sure if s2s_tunnel should be a usage field.  It looks like each tunnel is represented in terraform by the `azurerm_virtual_network_gateway_connection` resource.  I don't recall if there is a way to gather up all the subresources so we can tell if there are more than 10 (they are priced at $0.15/hr for each tunnel over 10), but if not maybe we just treat each one as an independent resource with the $0.15/hr charge).  @alikhajeh1 do you have an opinion?

Reply:
removed s2s resource will create vpn_gateway_connection.go for vpn gateway connection resource